verification/client/abi.go

```diff
@@ -238,16 +244,14 @@ type DeriveContextReq[Digest DigestAlgorithm] struct {
 type DeriveContextResp struct {
 	NewContextHandle    ContextHandle
 	ParentContextHandle ContextHandle
+	ExportedCdi         ExportedCdi
+	CertificateSize     uint32
+	NewCertificate      []byte
```

Comment:
nit: should these be pointers so we can make them `nil` in the non-export case? (Note that Go pointers are refcounted, so we don't need to worry about memory management)

Reply:
All Go types have a default value, for example the `[]byte` will be filled with 0s.

I think my preference is to keep it as is because:
1. Would this impact serialization (Acknowledging that I added anonymous types to deserialize).
2. I tried switching to pointers and couldn't make the type checker happy but I'm sure with more elbow grease I could find something that works.